core/modules/widgets/dropzone.js

```diff
@@ -27,14 +27,14 @@ DropZoneWidget.prototype = new Widget();
 Render this widget into the DOM
 */
 DropZoneWidget.prototype.render = function(parent,nextSibling) {
-	var self = this;
```

Comment:
It looks like you're making a coding style update at the same time as making functional modifications. In general, I think it's better to make them separate requests, as having two things going on can make it harder to understand and verify a request. But I must stress I'm not against cleaning up of the wrinkles in the code; it's tricky work, though, as it's hard to systematically test such changes.